src/Compiler/Microsoft.AspNetCore.Razor.Language/src/Microsoft.AspNetCore.Razor.Language.csproj

```diff
@@ -8,4 +8,8 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\Razor\src\Microsoft.AspNetCore.Razor.Common\Microsoft.AspNetCore.Razor.Common.csproj" />
```

Comment:
Why did this reference become necessary?

Reply:
This is to do with the nullability attributes. The compiler had its own copy, which was conflicting in downstream projects with the ones in common. I removed it from here and added the reference here instead.